apiary.apib

```diff
@@ -64,17 +64,174 @@ Any response that does not provide a ```200 OK``` response code should be treate
 
 + Response 200 (application/json)
     + Headers
-    
+
             X-Dude-Api-Limit: Unlimited or Number
             X-Dude-Api-Remaining: Unlimited or Number
             X-Dude-Api-Server: Api Server ID
-            
+
     + Attributes (Heartbeat)
 
+# Group Master Data
+The master data collection allows for integration between the Dude Platform and the dudes other system of records.
+
+## GetUser [/masterdata/users/{userName}]
++ Parameters
+    + userName (string, required) - The name of the user from the system of record.
+
+### GetUser [GET]
++ Response 200 (application/json)
+    + Attributes (UserDetail)
+
+## GetAccount [/masterdata/accounts/{accountNumber}]
++ Parameters
+    + accountNumber (string, required) - The account number for the system of record account.
+
+### GetAccount [GET]
++ Response 200 (application/json)
+    + Attributes (Account)
+
+## GetProduct [/masterdata/products/{productId}]
++ Parameters
+    + productId (number, required) - The product Id from the system of record.
+
+### GetProduct [GET]
++ Response 200 (application/json)
+    + Attributes (Asset)
+
+## Add User System of Record Map [/masterdata/{sourceAbbreviation}/users/{userName}/map/{userSummaryId}]
++ Parameters
+    + sourceAbbreviation (string, required) - The system of record source abbreviation.
+    + userName (string, required) - The user name from the system of record.
+    + userSummaryId (number, required) - The dude platform user to map too.
+
+### Add User System of Record Map [POST]
++ Response 201 (application/json)
+    + Attributes (SystemOfRecordUser)
+
 # Data Structures
 
 ## Heartbeat (object)
 Represents a heartbeat response.
 
 ### Properties
-+ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
\ No newline at end of file
++ `vitalCheck`: true (boolean, required) - Vital check to ensure something is online.
+
+## UserDetail (object)
+Represents a UserDetail object
+
+### Properties
```

Comment:
This is the proper format when the object contains a description directly underneath it.  The explicit notation is fine.